challenges/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,11 @@
+const arrayReverse = function(array) {
+    let newArray = array;
+    for(let i = newArray.length - 1; i >= (newArray.length -1) / 2; i--) {
```

Comment:
this was a little hard for me to follow. here's a less verbose way to do it.
function reverse(a) {
	const newArr = [];
	for(let i = 0; i < a.length; i++) {
	newArr[i] = a[a.length - (i + 1)];
    }
return newArr;
}